cpp/tensorrt_llm/common/attentionOp.cpp

```diff
@@ -1651,6 +1651,11 @@ int AttentionOp::enqueueContext(EnqueueContextParams<T> const& params, cudaStrea
         fmhaParams.stream = stream;
         fmhaParams.forceFp32Acc = mFMHAForceFP32Acc;
 
+        if (mAttentionChunkSize)
+        {
+            fmhaParams.chunkedAttentionSize = *mAttentionChunkSize;
```

Comment:
Not all attention backends in AttentionOp respect this argument. Please add proper warnings/error messages.